src/lib/ListMetrics.js

```diff
@@ -257,6 +257,7 @@ export default class ListMetrics {
                     const m = getMetricFromKey(key);
                     let count = parseInt(item[1], 10);
                     count = Number.isNaN(count) ? 0 : count;
+                    count = count < 0 ? 0 : count;
```

Comment:
Since this affects only `storageUtilized` and `numberOfObjects`, can you add a comment saying so. It will be easy for maintenance.